nuxt.config.ts

```diff
@@ -48,7 +48,22 @@ export default defineNuxtConfig({
   },
   app: {
     pageTransition: { name: "page", mode: "out-in" },
-    head: { base: { href: "/" } },
+    head: {
+      base: { href: "/" },
+
+      link: [
+        { href: "https://fonts.googleapis.com", rel: "preconnect" },
+        {
+          href: "https://fonts.gstatic.com",
+          rel: "preconnect",
+          crossorigin: "anonymous",
+        },
+        {
+          href: "https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700;800&display=swap",
```

Comment:
I'm pretty sure we use weight 400 somewhere. Maybe 900 as well.

Reply:
The design system doesn't include those, so that's kinda wrong. but sure, no biggie to include those. done.

Reply:
I was wrong. They aren't used